From ddfb264372197d3e0b0d356038ac42e204d2733d Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 23 Oct 2017 20:46:04 +0800
Subject: [PATCH 2677/5242] MLK-16681-2 ARM64: dts: freescale: imx8mq: update
 cpu-freq set-points

commit  98a4c451966fd5a3be8307dfc33f8e7c5b97f1d9 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update cpu-freq set-points according to datasheet Rev-E:

		Normal          Over-Drive
Consumer	1GHz@0.9V	1.5GHz@1V
Industrial	800MHz@0.9V	1.3GHz@1V

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm64/boot/dts/freescale/fsl-imx8mq-evk-od.dts |    7 ++++---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi      |    2 +-
 2 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-od.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-od.dts
index ac1b658..3722a3b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-od.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-od.dts
@@ -42,9 +42,10 @@
 &A53_0 {
 	operating-points = <
 		/* kHz    uV */
-		1400000 1000000
-		1200000 900000
-		800000	900000
+		1500000 1000000
+		1300000 1000000
+		1000000 900000
+		800000  900000
 	>;
 	dc-supply = <&reg_gpio_dvfs>;
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index d90151d..f00abd2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -1108,7 +1108,7 @@
 &A53_0 {
 	operating-points = <
 		/* kHz    uV */
-		1200000 900000
+		1000000 900000
 		800000	900000
 	>;
 	clocks = <&clk IMX8MQ_CLK_A53_DIV>, <&clk IMX8MQ_CLK_A53_SRC>,
-- 
1.7.9.5

